js/models/ServerConfig.js

```diff
@@ -56,9 +54,8 @@ export default class extends BaseModel {
       ...attrs,
     };
 
-    if (fullAttrs.builtIn && typeof attrs.walletCurrency === 'string') {
-      attrs.name = app.polyglot.t('connectionManagement.builtInServerName',
-        { cur: attrs.walletCurrency });
+    if (!this.get('builtIn') && fullAttrs.builtIn) {
```

Comment:
> You're only setting the name if the model's builtIn is false and is being set to true, is the intent to avoid setting the name if it's already been set?

Yes, I thought it would potentially be slightly more efficient to only set it if it needs settings instead of just always setting it and sending it off through BB's plumbing.

But... I was on the fence, not sure if I was overthinking, so I just simplified to:

```
    if (fullAttrs.builtIn) {
      attrs.name = app.polyglot.t('connectionManagement.builtInServerName');
    }
```